### PR TITLE
URL encode in email verification token

### PR DESCRIPTION
**Pull request by skystar-p**

### CODE REVIEW

File: backend/api/sign_views.py [hunk 1]
```diff
@@ -69,7 +70,7 @@ def sign_up(request):
         code = str(uuid.uuid4())
         cache.set('verify-account:' + username, code)
 
-        url_code = b64encode(':'.join((username, code)).encode()).decode()
+        url_code = quote(b64encode(':'.join((username, code)).encode()).decode(), safe='')
```
Comment (tirr-c) [inline backend/api/sign_views.py]: `safe=''`를 지우고 `quote_plus`를 쓰는 게 좋을 것 같다

File: backend/api/tests.py [hunk 2]
```diff
@@ -282,7 +283,7 @@ def test_sign_up_success(self):
         link = body.split('link.')[1].strip()
 
         post_data = {
-            'token': link.split('/')[-2].strip(),
+            'token': unquote(link.split('/')[-2].strip()),
```
Comment (tirr-c) [inline backend/api/tests.py]: `unquote_plus`

Comment (skystar-p) [reply backend/api/tests.py]: 필요 없긴 한데... 테스트라서..

**Comment by tirr-c** [comment]
영 찝찝하긴 하지만 @web-mighty-homu r+

**Comment by web-mighty-homu** [comment]
:pushpin: Commit da88bfe has been approved by `tirr-c`

**Comment by web-mighty-homu** [comment]
:hourglass: Testing commit da88bfe5a87cbe79ed5d1037a3439762db920437 with merge c4a5b97e0ca79c207ec26ca37100b3ed9553e571...

**Comment by tirr-c** [comment]
what @web-mighty-homu r-

**Comment by tirr-c** [comment]
@web-mighty-homu r+

**Comment by web-mighty-homu** [comment]
:pushpin: Commit de70a87 has been approved by `tirr-c`

**Comment by web-mighty-homu** [comment]
:hourglass: Testing commit de70a875f516648747a86d99a697fdbe88dc6fc8 with merge 58767968a89e5cb1b4fdfb4796846647808bb04d...

**Comment by web-mighty-homu** [comment]
:sunny: Test successful - [status-travis](https://travis-ci.org/swsnu/swpp17-team2/builds/312263342?utm_source=github_status&utm_medium=notification)
Approved by: tirr-c
Pushing 58767968a89e5cb1b4fdfb4796846647808bb04d to master...

**Pull request merged**